src/components/Home/Ritika/styles.ts

```diff
@@ -1,3 +1,14 @@
 import styled from 'styled-components';
 
-export const RitikaWrapper = styled.div``;
+export const MainContainerWrapper = styled.div`
+  .weekButton {
```

Comment:
use same format while defining classnames

Reply:
i thought that for we need to define it in uppercamel case